views/surveys/view_survey_results.php

```diff
@@ -0,0 +1,259 @@
+<?php
+global $params, $route, $view;
+include ('../models/Notification.php');
+$pages = ['view','edit','create','delete','restore'];
+# Update page title to reflect route
+if (!empty($params) && in_array($params[0], $pages)) {
+	$newTitle = $params[0];
+	$route['title'] .= ' - ' . strtoupper($newTitle[0]) . strtolower(substr($newTitle, 1));
+}
+# Select page to display
+if (!empty($params) && $params[0] == 'view') {
+	$view->display('/view-survey-results.php');
+} else if (!empty($params) && $params[0] == 'results') {
+    $view->display('surveys/results.php');
+} else {
+	include('header.php');
+	global $db;
+	$filter = "";
+	
+	if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
Are you ever using a filter on this page? This purpose of this if statement is the filter you see on the curriculum page at the top. You don't need this whole if-statement.